pkg/templates/templates/credentials/create/credential-set.schema.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
The file name is still confusing because it has "schema" in it (but this isn't a schema). Can you please rename it to "credential-set.json"